src/utils/libraryChecker.ts

```diff
@@ -0,0 +1,23 @@
+import * as child_process from 'child_process';
```

Comment:
IMO a new file is not required and this function can be added in `utils/misc.ts` file itself

Reply:
Yes. That is what I was planning too. makes more sense to have it in `utils/misc.ts`.

Reply:
Piling up random bits of code in modules with not clearly specified purpose is an antipattern. It's `misc` (no visible purpose in the name) and it's under `utils` (nothing specific either). The structure should reflect what it's for.

Reply:
Maybe, we can have an elaborated discussion on this in the team calls @webknjaz, @ganeshrn.